tests/test_extract_lyrics.py

```diff
@@ -0,0 +1,65 @@
+from pylyrics import extract_lyrics as pl
+import json, os
+import pytest
+
+# add token file to "/data/credentials.json"
+# {
+#   "token": "YOUR-TOKEN"
+# }
+
+
+def get_token():
+    current_path = os.getcwd()
+    with open(current_path + "/tests/data/credentials.json") as f:
+        login = json.load(f)
+    return login["token"]
+
+
+# Case 1 - happy case
+def test_happy_case():
+    token = get_token()
+    current_path = os.getcwd()
+    arr_happy = ["22", "Taylor Swift"]
+    target = open(current_path + "/tests/data/lyrics_22.txt", "r").read()
+    assert pl.get_lyrics(token, arr_happy[0], arr_happy[1]) == target, "Lyrics output incorrect"
+  
+
+# Case 2 - empty dataframe
+def test_empty_case():
+    try:
+        token = get_token()
+        arr_edge1 = ["", "Taylor Swift"]
+        with pytest.raises(ValueError):
+            pl.get_lyrics(token, arr_edge1[0], arr_edge1[1])
+        assert pl.get_lyrics(token, arr_edge1[0], arr_edge1[1]) == None, "Edge case exception handling failed, should return None"    
+    except:
+        assert True
```

Comment:
Do we need the try/except for the last couple of functions?

Reply:
I have try-except on this function?
please clarify on the meeting, thanks!